extensions/wikia/Scribe/ScribeEventProducer.class.php

```diff
@@ -126,30 +165,53 @@ public function buildEditPackage( $oPage, $oUser, $oRevision = null, $oLocalFile
 		return true;
 	}
 
-	public function buildRemovePackage ( $oPage, $oUser, $page_id ) {
-		wfProfileIn( __METHOD__ );
+	/**
+	 * @param $oPage WikiPage
+	 * @param $oUser User
+	 * @param $page_id int
+	 * @return bool|int
+	 */
+	public function buildRemovePackage( $oPage, $oUser, $page_id ) {
+		global $wgCityId;
 
-		if ( !is_object( $oPage ) ) {
-			Wikia::log( __METHOD__, "error", "Cannot send log using scribe ({$this->app->wg->CityId}): invalid WikiPage object" );
-			wfProfileOut( __METHOD__ );
-			return false;
-		}
+		wfProfileIn( __METHOD__ );
 
-		if ( !$oUser instanceof User ) {
-			Wikia::log( __METHOD__, "error", "Cannot send log using scribe ({$this->app->wg->CityId}): invalid user object" );
+		if ( !( $this->isObject( $oPage ) && $this->isUser( $oUser ) ) ) {
 			wfProfileOut( __METHOD__ );
 			return false;
 		}
 
 		$oTitle = $oPage->getTitle();
 
+		if ( $oTitle instanceof Title && $oTitle->getNamespace() == NS_FILE ) {
```

Comment:
See the other comment (https://github.com/Wikia/app/pull/11185/files#diff-b0907bc8578392048108d742c3e2fe83R148)